pages/data-migration/csv.mdx

```diff
@@ -180,8 +180,14 @@ You can switch between modes within the session using the following query:
 STORAGE MODE IN_MEMORY_{TRANSACTIONAL|ANALYTICAL};
 ```
 
-When in the analytical storage mode, **don't** import data using multiple
-threads.
+If you use `IN_MEMORY_ANALYTICAL` mode and have nodes and relationships stored in
+ separate CSV files, you can run multiple concurrent `LOAD CSV` queries to import data even faster.
+In order to achieve the best import performance, split your nodes and edge
+files into smaller files and run multiple `LOAD CSV` queries in parallel. 
+The key is to run all `LOAD CSV` queries for nodes first. After that, run 
+all relationship `LOAD CSV` queries. 
+
+Read more about this in [import best pratices](TBD) 
```

Comment:
```suggestion
Read more about this in the [import best practices](./best-practices) 
```

Reply:
we'll update it later if needed

Reply:
Let's updated the readme there and results, then we will just add links to places.